ci/docker/snaper/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# build image: docker build --no-cache -f ci/docker/snaper/Dockerfile --tag ghcr.io/nordsecurity/nordvpn-linux/snaper:0.0.1 .
```

Comment:
wouldn't be good, maybe in another PR, to verify the downloaded packages?

Reply:
for example for OpenVPN, `sha256sum -c` is being used to check the downloaded file.
There is also some `download_sha512` included into the JSON response, e.g. `https://api.snapcraft.io/api/v1/snaps/details/core`.

Is this check needed also in this case?

Reply:
TBH, doubt that...